hal/src/stm32f2xx/rtc_hal.c

```diff
@@ -84,21 +101,187 @@ void HAL_RTC_Initialize_UnixTime()
     /* Get calendar_time date struct values */
     RTC_DateStructure.RTC_WeekDay = 6;
     RTC_DateStructure.RTC_Date = 1;
-    RTC_DateStructure.RTC_Month = 0;
+    RTC_DateStructure.RTC_Month = 1;
```

Comment:
Just to double confirm that the month starts from 1, since the month field in some RTC chip varies from 0 ~ 11, which is corresponding to month 1 ~ 12.

Reply:
Here, it's actually expected to start with 1. Previously the date ended up being Dec 1999 instead of Jan 2000.